### ISSUE TITLE
Bug: cannot remove/clear document title

### ISSUE THREAD

**Issue by JakeSidSmith** [nfl/react-helmet#710]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

Attempting to remove a title with any of the following does not remove the title: `<title>{''}</title>`, `<title>{undefined}</title>`, `<title />`

Minimal reproduction: https://codepen.io/jakesidsmith/pen/abPLPxm

**What is the expected behavior?**

The document title should be removed (set to empty string).

**Which versions of React and react-helmet, and which browser / OS are affected by this issue? 
Did this work in previous versions of React and/or react-helmet?**

```
react@18.2.0
react-helmet@6.1.0
```

I have not tested any other versions.

Additional notes:

This is a particularly frustrating limitation as I intend to use react-helmet within one of my libraries, so I won't have control over users' code, but I want users of my library to be able to see where they have forgotten to add a title and currently (because react-helmet maintains the previous title) it is hard to know where titles have been missed. To make things worse, only after reloading the page, or a user navigating to a page for the first time will the missing title be highlighted.

**Comment by JakeSidSmith** [comment]
I opened an issue rather than a pull request, as I'm not sure if this is intended behavior.

If this is intended, then it would be nice to have an option to disable it. I'd be happy to add this feature myself.

**Comment by moon-xt** [comment]
same issue

**Comment by zychen31** [comment]
same issue
